apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/service/PersonService.java

```diff
@@ -0,0 +1,31 @@
+package no.nav.registre.skd.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.registre.skd.consumer.PersonConsumer;
+import no.nav.registre.testnorge.dto.person.v1.PersonDTO;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class PersonService {
+
+    private final PersonConsumer personConsumer;
+
+    public void leggTilIdenterIPdl(List<Long> identer) {
```

Comment:
Har kommentert dette i klassen SyntetiseringService, men det er altså en annen liste som er av type `List<String>` som skal inn her. 

Sånn generelt er det red alert hvis en liste av fødselsnumre er av type Long, eller noe annet enn String, for da mister man ledende 0-er. (f.eks vil et fødselsnummer 05069012345 bli 5069012345, som ikke validerer som et fødselsnummer)

Reply:
Skjønner, skal unngå det i fremtiden :)